src/app/clusters/power-source-server/power-source-server.cpp

```diff
@@ -34,10 +35,55 @@ using namespace app::Clusters::PowerSource::Attributes;
 
 namespace {
 
+struct PowerSourceClusterInfo
+{
+    PowerSourceClusterInfo() : mClusterEndpoint(kInvalidEndpointId) {}
+    explicit PowerSourceClusterInfo(EndpointId powerClusterEndpointId) : mClusterEndpoint(powerClusterEndpointId) {}
+    void Shutdown()
```

Comment:
Suggest rename to `clear()`

Reply:
done.

Reply:
oop, or it will be done once I push the next commit.